src/components/podcast/Podcast.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Episode from './episode/Episode';
+import PodcastHeader from './Header'
+import parsePodcastFeedFromXML from '../../util/podcast-util';
+
+/**
+ * Component for displaying the details about a podcast including the episodes
+ */
+const Podcast = (props) => {
+
+    // Track the podcast object
+    const [podcast, setPodcast] = useState({});
+    // Track the episodes separately
+    const [episodes, setEpisodes] = useState([])
+    // Track if the podcast has been loaded
+    const [loaded, setLoaded] = useState(false);
+
+    useEffect(() => { loadFeed(props.feed + '.xml') });
```

Comment:
Perfect. You might also want to add a second parameter of `[]` to useEffect so this podcast is only loaded when this component is mounted.

Reply:
I _think_ that'd get rid of the need for the `loaded` state... but practically speaking you'd probably want to keep track of whether it's loading so you can display a spinner or something